### PR TITLE
NEW Mass action close shipment

### PR DESCRIPTION
**Pull request by 1ocate**
If Shipment so many wasting time for close.

This make close check list.

### CODE REVIEW

File: htdocs/expedition/list.php [hunk 1]
```diff
@@ -473,7 +501,7 @@
 
 $arrayofmassactions = array(
 	'builddoc' => img_picto('', 'pdf', 'class="pictofixedwidth"').$langs->trans("PDFMerge"),
-	//'classifyclose'=>$langs->trans("Close"), TODO massive close shipment ie: when truck is charged
+	'classifyclose'=>$langs->trans("Close"), TODO massive close shipment ie: when truck is charged
```
Comment (frederic34) [inline htdocs/expedition/list.php:504]: missing // before TODO

Comment (1ocate) [reply htdocs/expedition/list.php:504]: Hi @frederic34 That's right.

Thank you for fix.

I'm still begineer.

Comment (1ocate) [reply htdocs/expedition/list.php]: @frederic34 
I think "TODO massive close shipment ie: when truck is charged" should remove.. right?

**Comment by 1ocate** [comment]
@eldy 
hi 

Thank you for make useful open source ERP.

I start use dolibarr since 2019 and my company also more stable with dolibarr.

And if I can I want  bug fix or add something..

My english is bad and develop skill not good but I want  try and keep going..

If I have mistake please forgive.. and please let know I will fallow rules.

Thank you reading this and great job.

**Review by 1ocate — COMMENTED** [review]
Hi. 

This is right? need more change?

**Pull request merged**